command/server.go

```diff
@@ -937,13 +937,40 @@ CLUSTER_SYNTHESIS_COMPLETE:
 		return 1
 	}
 
+	// Attempt an initial unseal.
 	if err := core.UnsealWithStoredKeys(context.Background()); err != nil {
```

Comment:
Not sure what to do here. We already have code that attempted an unseal with stored keys. The previous method would return `nil` if unseal keys aren't supported. Now it returns a fatal error, so I think we need to wrap these calls with a check to see if unseal keys are supported. Additionally, I'm not sure if we need this block anyway since the goroutine will do it later. The only difference is that the goroutine won't force the CLI to exit on error (it'll just log).